samples/pipelines/sklearn/params.yaml

```diff
@@ -1,2 +1,2 @@
 eval_acc_threshold: 0.87
-start_date: "{% now 'utc', '%Y/%m/%d' %}"
+start_date: "{{ modules.pendulum.now().subtract(days=1).format('YYYY/MM/DD') }}"
```

Comment:
Is it okay that this one does not have the `.in_timezone('UTC')` ?

Reply:
yes it is ok. Is just an example and the users may or may not use `.in_timezone('UTC')`